src/SecretLobby.Tests.Core/Tests/GettingServers.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Linq;
+using static SecretLobby.Tests.Core.Program;
+
+namespace SecretLobby.Tests.Core.Tests
+{
+    public class GettingServers : ITest
+    {
+        public void Invoke()
+        {
+            ConsoleLog("Getting servers...");
```

Comment:
Testing this entire project isn't really feasible, this is an integration test rather than testing any logic.

Reply:
I created this with an estimate for the future, it will also change its appearance, because I plan to get rid of static methods of accessing the API